### PR TITLE
[SYCLomatic] [DOC] [Experimental] Add a new option in doc

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

File: docs/dev_guide/command-line-options-reference.rst [hunk 1]
```diff
@@ -38,6 +38,9 @@ in alphabetical order.
    * - ``=c_cxx_standard_library``
      - A set of functions from the C and C++ standard libraries that are allowed
        to be used in SYCL device code.
+   * - ``--change-filename-extension=<file>``
+     - DEPRECATED: Specifies which extensions of migrated fileswill be changed,
```
Comment (zhimingwang36) [inline docs/dev_guide/command-line-options-reference.rst]:
```suggestion
     - DEPRECATED: Specifies which extension of migrated files will be changed,

```

File: docs/dev_guide/command-line-options-reference.rst [hunk 2]
```diff
@@ -38,6 +38,9 @@ in alphabetical order.
    * - ``=c_cxx_standard_library``
      - A set of functions from the C and C++ standard libraries that are allowed
        to be used in SYCL device code.
+   * - ``--change-filename-extension=<file>``
+     - DEPRECATED: Specifies which extension(s) of migrated file(s) will be changed,
```
Comment (krisdale) [inline docs/dev_guide/command-line-options-reference.rst]: "DEPRECATED: A comma-separated list that specifies which extensions of migrated files will be changed. Default..."

**Review by krisdale — CHANGES_REQUESTED** [review]
Approved if the changes are applied.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**